src/modules/iotjs_module_mqtt.c

```diff
@@ -94,6 +94,22 @@ int iotjs_mqtt_read_buf(unsigned char* buf, int len, void* ctx) {
   return len;
 }
 
+static void iotjs_mqtt_alloc_buf(unsigned char **buf, int expected_size, int *alloc_size) {
+  static int buf_size = 128 * 1024;
+  static unsigned char *buf_ = NULL;
+  if (buf_ == NULL) {
+    buf_ = malloc((size_t)buf_size * sizeof(unsigned char));
+    if (buf_ == NULL) {
+      return;
```

Comment:
We should throw an OOM error to abort the whole process.

Reply:
should we just abort the program or exit with some code?

Reply:
Let's ignore this one for this time.